README.md

```diff
@@ -31,6 +31,11 @@ Usage
 ```
 npm run desktop
 ```
+[Dowload mqtt.zip](https://github.com/asyncapi/simulator/files/13696873/mqtt.zip)
```

Comment:
Can you put and explanation of what this file is.

Like: Example asyncApi file used in the video

Reply:
Done !! :tada: